src/test/kotlin/vonage/spring/VerifiedSmsHandlerTest.kt

```diff
@@ -0,0 +1,55 @@
+package vonage.spring
+import com.vonage.client.auth.RequestSigning
+import com.vonage.client.auth.hashutils.HashUtil
+import org.apache.http.NameValuePair
+import org.apache.http.message.BasicNameValuePair
+import org.junit.Test
+import org.springframework.mock.web.MockHttpServletRequest
+import org.springframework.mock.web.MockHttpServletResponse
+import java.time.Instant
+import java.util.HashMap
+import javax.servlet.http.HttpServletRequest
+import javax.servlet.http.HttpServletResponse
+import kotlin.test.assertEquals
+import kotlin.test.assertFalse
+import kotlin.test.assertTrue
+
+class VerifiedSmsHandlerTest {
+
+    private fun constructDummyRequest() : HttpServletRequest
+    {
+        val request = MockHttpServletRequest()
+        val params = HashMap<String, Array<String>>()
+        val nameValuePair = ArrayList<NameValuePair>(3)
```

Comment:
Unless there is a strict need for performance usually use `List` or `MutableList` in Kotlin